src/backends/plonky2/mock_main/mod.rs

```diff
@@ -26,14 +28,14 @@ impl PodProver for MockProver {
     }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct MockMainPod {
     params: Params,
     id: PodId,
-    input_signed_pods: Vec<Box<dyn Pod>>,
-    input_main_pods: Vec<Box<dyn Pod>>,
+    //   input_signed_pods: Vec<Box<dyn Pod>>,
+    //   input_main_pods: Vec<Box<dyn Pod>>,
```

Comment:
These were not being used anywhere, so I presume it's safe to comment them out and not worry about serializing them. This might be a bad idea, in which case I'm happy to change it.